wrappers/java/src/test/java/org/hyperledger/indy/sdk/metrics/CollectMetricsTest.java

```diff
@@ -0,0 +1,16 @@
+package org.hyperledger.indy.sdk.metrics;
+
+import org.hyperledger.indy.sdk.IndyIntegrationTest;
+import org.junit.Test;
+import static org.junit.Assert.assertNotNull;
+
+
+public class CollectMetricsTest extends IndyIntegrationTest {
+
+	@Test
+	public void testCollectMetricsMethod() throws Exception {
+		String metrics_map = Metrics.collectMetrics().get();
+		assertNotNull(metrics_map);
```

Comment:
I would also suggest to test that returned string is a valid JSON

Reply:
Added a few checks for a result JSON.